languages/js/src/Oso.test.ts

```diff
@@ -1,8 +1,29 @@
 import { Oso } from './Oso';
+import { InlineQueryFailedError } from './errors';
 
 test('#isAllowed', () => {
   const o = new Oso();
   o.loadStr('allow(1, 2, 3);');
   expect(o.isAllowed(1, 2, 3)).toBe(true);
   expect(o.isAllowed(3, 2, 1)).toBe(false);
 });
+
+describe('Equality function used for unification', () => {
+  test('defaults to loose equality (==)', () => {
+    const o = new Oso();
+    o.registerConstant('undefined', undefined);
+    o.registerConstant('null', null);
+    expect(() => o.loadStr('?= undefined = null;')).not.toThrow(
```

Comment:
Nit: can we please use `query` instead of inline queries? I find it much clearer to separate the two, and would prefer that we only use inline queries in policy files.